Source/Core/DolphinWX/GameListCtrl.h

```diff
@@ -107,6 +107,8 @@ class CGameListCtrl : public wxListCtrl
   static bool MultiCompressCB(const std::string& text, float percent, void* arg);
   static bool WiiCompressWarning();
 
+  struct ColumnInfo;
```

Comment:
This should be placed before functions, since it's a type definition (hence why I included `void ReloadList();`, the first function, as the reference point in my example).